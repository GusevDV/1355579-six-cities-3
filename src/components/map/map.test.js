import React from "react";
import renderer from "react-test-renderer";
import Map from "./Map.jsx";

const Offers = [
  {
    id: 1,
    title: `Apartment 1`,
    price: 100,
    pricePeriod: `night`,
    thumnnailUrl: `/img/apartment-01.jpg`,
    objectType: `apartment`,
    isPremium: false,
    rating: 50,
    cityId: 1,
    coords: [52.3909553943508, 4.85309666406198]
  },
  {
    id: 2,
    title: `Apartment 2`,
    price: 200,
    pricePeriod: `night`,
    thumnnailUrl: `/img/apartment-02.jpg`,
    objectType: `apartment`,
    isPremium: true,
    rating: 80,
    cityId: 1,
    coords: [52.3909553943508, 4.85309666406198]
  }
];

it(`Should Map component render correctly`, () => {
  const div = global.document.createElement(`div`);
  global.document.body.appendChild(div);
  const tree = renderer
    .create(<Map
      city={[52.38333, 4.9]}
      offers={Offers}
    />,
    {attachTo: div
    })
    .toJSON();

  expect(tree.getDOMNode()).toMatchSnapshot();
});
