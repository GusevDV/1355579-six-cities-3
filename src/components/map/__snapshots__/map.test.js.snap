// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Map component render correctly 1`] = `
<section
  className="cities__map map"
  id="map"
/>
`;
